content/platforms/kubernetes/reference/redis_enterprise_cluster_api.md

```diff
@@ -0,0 +1,180 @@
+---
+Title: Redis Enterprise Cluster API
```

Comment:
```suggestion
Title: Redis Enterprise Cluster - Custom Resource Specification
```

Reply:
This document is a more complete version of "cluster_options.md". We should not have both.

Reply:
This is a more complete version of cluster-options.md and we should not include both.